css/amp.css

```diff
@@ -165,6 +165,17 @@ html.i-amphtml-ios-embed.i-amphtml-ios-overscroll > #i-amphtml-wrapper {
   position: relative;
 }
 
+.i-amphtml-layout-intrinsic
```

Comment:
needs an additional:
```
[layout=intrinsic][width][height]:not(.i-amphtml-layout-intrinsic),
```
if JS fails to load and `.i-amphtml-layout-intrinsic` is not applied

Reply:
adding